res/controllers/Traktor-Kontrol-S4-MK2-hid-scripts.js

```diff
@@ -1279,7 +1283,21 @@ TraktorS4MK2.callbackPregain = function(field) {
         } else if (delta < 0) {
             engine.setValue(group, "beats_translate_earlier", true);
         }
-    } else {
+    }
+    else if (TraktorS4MK2.controller.play_shift_pressed === 1) {
+        var bpm_sensitivity_multipler = 50;
+        if (delta > 0) {
+            for (var i = 0; i < bpm_sensitivity_multipler; i++){
+                engine.setValue(group, "beats_adjust_faster", true);
+            }
+
```

Comment:
extra newline

Reply:
oh sorry for the noise you fixed that in a later commit

Reply:
err wait this extra line is still in the diff of the whole branch but the others are fixed